contrib/for-tests/volumes/gluster/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:14.04
```

Comment:
1. You're going to need to include the standard copyright and license blurb at the head of every new file that you add.
2. Given that every other directory under for-tests/ follows the naming convention of xxx-tester, is there any reason not to follow that convention here?